server/codes.go

```diff
@@ -27,4 +27,7 @@ var (
 	CodeUnknownAttribute = "UNKNOWN_ATTRIBUTE"
 	// CodeUnknownError indicates something went wrong in unpredictable ways.
 	CodeUnknownError = "UNKNOWN_ERROR"
```

Comment:
You mean `CodeUnknownError`? getting rid of it means changing these too:

- https://github.com/giantswarm/microkit/blob/master/server/response_error.go#L33
- https://github.com/giantswarm/microkit/blob/master/server/spec.go#L53